stacks-signer/src/signer.rs

```diff
@@ -1438,6 +1480,42 @@ impl Signer {
         Ok(())
     }
 
+    /// Send DKG results to the miner coordinator so that it can coordinate signing rounds
+    fn send_dkg_results(&mut self, dkg_public_key: &Point) {
+        let Some(dkg_results_bytes) = self.pending_dkg_results.remove(dkg_public_key) else {
+            warn!("{self}: No DKG results found for key: {dkg_public_key}");
+            return;
+        };
+
+        if let Err(e) = self
+            .stackerdb
+            .send_message_bytes_with_retry(&MessageSlotID::DkgResults, dkg_results_bytes)
+        {
+            error!("{}: Failed to send DKGResults message to StackerDB, will continue operating.", self.signer_id;
+                       "error" => %e);
```

Comment:
Shouldn't this return an error?

Reply:
Right, this was just inherited from the code I broke out. Returning an error is better. I'll update.

Reply:
Updated.